rhc-ose-ansible/roles/openshift-provision/tasks/prerequisites.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: "Install required packages"
+  yum:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - wget
+    - git
+    - net-tools
+    - bind-utils
+    - firewalld
+    - iptables-services
+    - bridge-utils
+    - bash-completion
+    - atomic-openshift-utils
+    - vim-enhanced
+    - docker
+
+- name: "Update all packages (this can take a very log time)"
+  become: yes
+  yum: 
+    name: "*"
+    state: latest
+
+- name: "Disable iptables"
```

Comment:
Why disable iptables and enable firewalld? If anything, it should most likely be the other way around. Even so, I'm not sure anything needs/should be done related to any firewall. We used to disable firewalld, but that's no longer necessary as the openshift install does it now.